tests/view/observer/fakeselectionobserver.js

```diff
@@ -100,6 +100,60 @@ describe( 'FakeSelectionObserver', () => {
 		);
 	} );
 
+	it( 'should fire `selectionChangeDone` event after selection stop changing', ( done ) => {
+		// Note that it's difficult to test lodash#debounce with sinon fake timers.
+		// See: https://github.com/lodash/lodash/issues/304
+
+		const spy = sinon.spy();
+
+		viewDocument.on( 'selectionChangeDone', spy );
+
+		// Change selection.
+		changeFakeSelectionPressing( keyCodes.arrowdown );
+
+		// Wait 100ms.
+		setTimeout( () => {
```

Comment:
Hm... pity. Perhaps Sinon controls timeouts but not the `Date()` class. Or you'd need to initialize the fake timers before the `viewDocument` is created. Did your try doing this in `beforeEach()`?